middleware/README.md

```diff
@@ -0,0 +1,24 @@
+# Middleware
+
```

Comment:
It would be great to have a brief description of what middleware is in the context of this library.